### PR TITLE
Add gactions

### PR DESCRIPTION
**Pull request by dlackty**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/gactions.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "gactions" do
+  version "3"
+  sha256 :no_check # required as upstream package is updated in-place
+
+  url "https://dl.google.com/gactions/v#{version}/release/gactions-sdk_darwin.tar.gz"
```
Comment (vitorgalvao) [inline Casks/gactions.rb:5]: Lead me here from the homepage.

Comment (miccal) [reply Casks/gactions.rb:5]: A bit convoluted:

https://developers.google.com/assistant/conversational -->
Get Started -->
https://developers.google.com/assistant/conversational/overview -->
Quick start guide -->
https://developers.google.com/assistant/conversational/quickstart -->
Install the gactions command-line tool

**Comment by dlackty** [comment]
@miccal Thanks for updating the cask. I was not pretty sure if I should extract version variable.
@vitorgalvao Is there anything I should do at this moment? 

Thank you both again!

**Review by vitorgalvao — DISMISSED** [review]

**Comment by miccal** [comment]
Thank you @dlackty.

**Review by miccal — APPROVED** [review]

**Pull request merged**